Doc/library/functions.rst

```diff
@@ -134,9 +134,9 @@ are always available.  They are listed here in alphabetical order.
    This function drops you into the debugger at the call site.  Specifically,
    it calls :func:`sys.breakpointhook`, passing ``args`` and ``kws`` straight
    through.  By default, ``sys.breakpointhook()`` calls
-   :func:`pdb.set_trace()` expecting no arguments.  In this case, it is
-   purely a convenience function so you don't have to explicitly import
-   :mod:`pdb` or type as much code to enter the debugger.  However,
+   :func:`pdb.set_trace()` expecting an optional keyword argument `header`.  In
```

Comment:
I'm not a native english speaker, but "expecting an optional ..." doesn't feel right. 

I'm not sure if anything needs to change here, expect maybe to rephrase the current text to indicate that the default hook can be called without arguments.  Users can look up the documentation of pdb.set_trace for advanced usage.